packages/veritone-json-schemas/src/index.test.js

```diff
@@ -0,0 +1,65 @@
+import { verifyObject, verifyTranscript, VALIDATORS } from './index';
+import _ from 'lodash';
+
+import glob from 'glob';
+
+const categories = {};
+
+// dynamically import all json files in examples or invalid-examples
+const files = glob.sync("schemas/vtn-standard/**/**(examples|invalid-examples)/!(master.json)//*.json");
+for(let file of files) {
+  const categoryIndex = file.split("/").findIndex(directory => _.includes(directory, "examples")) - 1;
```

Comment:
Naming convention nitpick: `category` should be `validationContract` or `contract` in the context of this vtn-standard spec.  "Category" is an aiWARE-specific notion.